### ISSUE TITLE
[Bug Report][3.3.3] v-autocomplete input focus issues

### ISSUE THREAD

**Issue by tmclagen** [vuetifyjs/vuetify#17641]
### Environment
**Vuetify Version:** 3.3.3
**Vue Version:** 3.3.4
**Browsers:** Chrome 114.0.0.0
**OS:** Windows 11

### Steps to reproduce
1. Click on the select
2. Start typing
3. Navigate menu using keyboard arrow keys
4. Press enter to choose an item (optional)

### Expected Behavior
Search input retains focus throughout the whole interaction (even when navigating menu with arrows). After selecting an item (e.g. by pressing **Enter**) either the search input should be cleared OR its content should get selected WHILE keeping focus all the time.

### Actual Behavior
Search input loses focus after you used arrow keys and now you cannot continue typing and searching. The only way I managed to reach the search input using keys is by pressing **Esc** and closing the dropdown first. Which kind of works but still ruins the user flow. 

### Reproduction Link
[https://play.vuetifyjs.com/#...](https://play.vuetifyjs.com/#eNqlUj1v2zAU/CsEF7eAJQ7pZDhFgk4FCjRbB8uDQj3XbMUPPD4pCAz99zxScsw4S4BM0h2PvLtH7k7yPoR6HEBu5JbAhr4l+N44IbZj1YaQfzPQ3lFrHOBCzYqBvPa8CwjOtBBjZX0H/W0jI/SgCbqffHRs5EXSt49Z8OPofQRxwMHQG8HGpC2syN9yxQ49GXa8MPpoQjxDdY6srjMnZq60VUVVhlGjCSQi0DA3NjZ4JHESCAcxcTxvxYqntEpyNvQuJnlRTtwm7Zfd/utFkKPzwm5OtOJR97BaL+iBg3GcgvmNrft7gb/AeveK/nBatNAX1AO0WAJ9fEX3AY32NON9ajx35HZyLed6lW1D/S96x3d/Sjoedl7gcW9EZhLHtRNu5JEoxI1SunO8jS/YjFg7IOWCVXcsUzg4Mhaqztu7m/qm/qY6E6mka4i2ekT/FAH5kEYugbMNmcPzB6wWZemwUBU/q5gs3h2t2G8ErBBcBwj40UZX20rPq6V3zZL71LiJ590b9z9ejVrHPOblbXyycj5suepJTmuZhew8f/cvjgVEOw==)

**Comment by XieWeiJie** [comment]
I also encountered the same problem

**Comment by actionjacks** [comment]
Yes I have the same problem, I have no idea how to fix it without causing more errors.

**Comment by abdullah-336** [comment]
Yes I have also the same problem, Is there any update on it?
